test/test_katello/test_plugin.py

```diff
@@ -15,12 +15,12 @@
 import sys
 import httplib
 
-from unittest import TestCase
-
 from mock import patch, Mock
 
 from rhsm.connection import RemoteServerException
 
+from unittest import TestCase
```

Comment:
Is this move needed?

Reply:
No, but I gave in to some of the flake8 whingings